app/uk/gov/hmrc/lisaapi/models/AnnualReturn.scala

```diff
@@ -59,9 +60,43 @@ object AnnualReturn {
     (JsPath \ "marketValueStocksAndShares").read(JsonReads.annualFigures) and
     (JsPath \ "annualSubsCash").read(JsonReads.annualFigures) and
     (JsPath \ "annualSubsStocksAndShares").read(JsonReads.annualFigures) and
-    (JsPath \ "supersede").readNullable[AnnualReturnSupersede]
+    (JsPath \ "supersede").readNullable[AnnualReturnSupersede] and
+    Reads.pure[Option[LifeEventId]](None)
   )(AnnualReturn.apply _)
 
+  val desReads: Reads[AnnualReturn] = (
```

Comment:
Is there something I've missed for why the Reads for an AnnualReturn ended up in this PR?